test/jasmine/tests/gl2d_click_test.js

```diff
@@ -246,12 +246,20 @@ describe('Test hover and click interactions', function() {
             pointNumber: 0
         });
 
+        // after the restyle, autorange changes the y range
```

Comment:
For some reason I don't understand, when this test ran before my changes, the y-axis didn't get autoranged after the `restyle`, even though when I make the same `restyle` call on master in the test dashboard it does autorange. Anyway, now it *does* get autoranged during the test, as it should, which is why I needed to alter the `run` the second time around.